<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>697</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Input Directory</string>
     </property>
     <layout class="QFormLayout" name="formLayoutInputs">
      <item row="0" column="0">
       <widget class="QLabel" name="labelInput">
        <property name="text">
         <string>Directory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="inputLineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="browseInputButton">
        <property name="text">
         <string>Select Input Subject</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Output Directory</string>
     </property>
     <layout class="QFormLayout" name="formLayoutOutputs">
      <item row="0" column="0">
       <widget class="QLabel" name="labelOutput">
        <property name="text">
         <string>Directory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="outputLineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="browseOutputButton">
        <property name="text">
         <string>Select Output Subjects Directory</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="partsCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>In vivo</string>
     </property>
     <layout class="QFormLayout" name="formLayoutParts">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxInvivoParenchyma">
        <property name="text">
         <string>Parenchyma</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxInvivoAirways">
        <property name="text">
         <string>Airways</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxInvivoVascularTree">
        <property name="text">
         <string>Vascular Tree</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBoxInvivoLobes">
        <property name="text">
         <string>Lobes</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="partsCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Ex vivo</string>
     </property>
     <layout class="QFormLayout" name="formLayoutParts">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxExvivoParenchyma">
        <property name="text">
         <string>Parenchyma</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxExvivoAirways">
        <property name="text">
         <string>Airways</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonSegmentation">
     <property name="text">
      <string>Segment</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
